### PR TITLE
Order page

### PR DESCRIPTION
**Pull request by QaisSaada**
# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by Sana-Shabeel — APPROVED** [review]

File: packages/client/src/components/Order-Page/OrderPage.component.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const fetchUser = () => {
+    fetch(`${getApiBaseUrl()}/api/orders`, {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' },
+    })
+      .then((res) => res.json())
+      .then((result) => {
+        setData(result);
+      });
+  };
```
Comment (miltonhultgren) [inline packages/client/src/components/Order-Page/OrderPage.component.js:24]:
```suggestion
  useEffect(() => {
  async function fetchUser() {
    const response = await fetch(`${getApiBaseUrl()}/api/orders`, {
      method: 'GET',
      headers: { 'Content-Type': 'application/json' },
    });
    const result = await response.json();
    setData(result);
  };
  
    fetchUser();
  }, []);
```

What about loading state and error handling?

File: packages/client/src/components/Order-Page/OrderPage.component.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  return (
+    <div>
+      <div className="order-page">
+        <p className="order-page-path">
+          Simply Spices / Shopping Cart and Delivery Details{' '}
```
Comment (miltonhultgren) [inline packages/client/src/components/Order-Page/OrderPage.component.js:30]:
```suggestion
          Simply Spices / Shopping Cart and Delivery Details
```

Use CSS margin instead

File: packages/client/src/components/Order-Page/OrderPage.component.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        </p>
+        <h1 className="order-page-title">Shopping Cart</h1>
```
Comment (miltonhultgren) [inline packages/client/src/components/Order-Page/OrderPage.component.js:32]: A page with multiple `h1`s is an error. If you want the text style, then use a div and add a CSS class instead. `h1` means something very specific.

**Review by miltonhultgren — CHANGES_REQUESTED** [review]

File: packages/client/src/components/Order-Page/OrderPage.component.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import ShoppingCart from '../ShoppingCart/ShoppingCart.component';
+import DeliveryDetails from '../DeliveryDetails/DeliveryDetails.component';
+import Carousel from '../Carousel/Carousel.component';
+import getApiBaseUrl from '../../utils/getApiBaseUrl';
+import './OrderPage.css';
+
+function OrderPage() {
+  const [data, setData] = useState([]);
```
[the remark this answers is not in the document]
Comment (QaisSaada) [reply packages/client/src/components/Order-Page/OrderPage.component.js:9]: the best kind!

File: packages/client/src/components/ShoppingCart/ShoppingCart.component.js [hunk 5]
```diff
@@ -1,69 +1,65 @@
...
+
           return {
             ...product,
             quantity: product.quantity + 1,
           };
         }
-        return products;
+        return product;
       }),
     );
```
[the remark this answers is not in the document]
Comment (QaisSaada) [reply packages/client/src/components/ShoppingCart/ShoppingCart.component.js:25]: We don't understand this. Please Mind that we "freeze code" in 5 hours. 

This entire local-storage/context thing has not been a nice learning experience. We would highly appreciate a solution at this point and we will study it later. 

I apologies if this reads as a dry comment ':)

[the remark this answers is not in the document]
Comment (miltonhultgren) [reply packages/client/src/components/ShoppingCart/ShoppingCart.component.js:25]: Leave it for now, I will send you some learning materials to reflect on!

**Review by miltonhultgren — APPROVED** [review]
I'm approving because the remaining changes aren't life critical and the critical stuff is a bit too much effort for the deadline coming.

**Pull request merged**